Maddori.Apple/Maddori.Apple/Screen/InProgress/InProgressViewController.swift

```diff
@@ -0,0 +1,197 @@
+//
+//  InProgressViewController.swift
+//  Maddori.Apple
+//
+//  Created by 이성민 on 2022/10/24.
+//
+
+import UIKit
+
+import SnapKit
+
+final class InProgressViewController: BaseViewController {
+    
+    private var keywordData = Keyword.mockData
+    private let currentRetrospective = "진저"
+    private let user = "이드"
+//    private let user = "진저"
+    private enum Size {
+        static let keywordLabelHeight: CGFloat = 50
+        static let sectionPadding: CGFloat = 60
+    }
+    private var keywords = [[Keyword]]()
+    private var isUserRetrospective: Bool {
+        return user == currentRetrospective ? true : false
+    }
+    
+    // MARK: - properties
+    
+    private let backButton = BackButton(type: .system)
+    private lazy var titleLabel: UILabel = {
+        let label = UILabel()
+        label.setTitleFont(text: currentRetrospective +  TextLiteral.InProgressViewControllerTitleLabel)
```

Comment:
p5;

```suggestion
        label.setTitleFont(text: currentRetrospective + TextLiteral.InProgressViewControllerTitleLabel)
```

아~~~~~주 사소한 띄어쓰기 한칸....

Reply:
[09ab5d5](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/59/commits/09ab5d5a288019ac16ca9eae963f0070a4b5d13c)
당신도 저 못지 않은 매의 눈이군요..